packages/back-end/test/services/dimensions.test.ts

```diff
@@ -0,0 +1,52 @@
+import { ExperimentDimensionWithSpecifiedSlices } from "shared/types/integrations";
+import {
+  countDimensionLevels,
+  trimPrecomputedDimensionsToMaxCells,
+} from "back-end/src/services/dimensions";
+
+describe("getPredefinedDimensionSlicesByExperiment", () => {
```

Comment:
nit: Should we update this name? `services/dimension` maybe?

Reply:
Sorry, I'm not sure I understand this question.